src/integrationtests/java/com/aws/greengrass/integrationtests/e2e/util/NetworkUtilsLinux.java

```diff
@@ -37,11 +39,15 @@ public void recoverNetwork() throws InterruptedException, IOException {
         modifyPolicy(disableOption, "connection-recover", NETWORK_PORTS);
     }
 
-    private void modifyPolicy(String option, String eventName, String... ports) throws InterruptedException,
-            IOException {
+    private void modifyPolicy(String option, String eventName, String... ports)
+            throws InterruptedException, IOException {
         for (String port : ports) {
-            logger.atWarn(eventName).kv("port", port).log(Platform.getInstance().createNewProcessRunner()
-                    .sh(String.format(commandFormat, option, port, option, port)));
+            logger.atWarn(eventName)
+                    .kv("port", port)
+                    .log(
+                            Platform.getInstance()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Ensure all untrusted input is properly sanitized before logging. Use parameterized logging or validate input against a allow list to prevent log injection vulnerabilities. Consider using a dedicated logging library&#39;s built-in sanitization features when available. Learn more - https://cwe.mitre.org/data/definitions/117.html